content/getting-started/signup.md

```diff
@@ -53,7 +53,7 @@ Here are some tips on what to check if you can't see your repository listed in C
 
 * If you chose to grant access to **select repositories only** as opposed to all repositories when setting up the GitHub App, make sure you have also granted access to the repository in question. Click **Manage integration > Configure** in team or user integrations to configure repository access settings.
 * Your personal Codemagic account and each team have separate integrations. Make sure you have connected the GitHub App integration in the right team or in your user settings.
-* If you build apps from different GitHub accounts or organizations, make sure that you have installed the GitHub App on the account that has the repository available. Click **Manage integration > Configure** to see which accounts have been connected. You can also visit [https://github.com/settings/installations](https://github.com/settings/installations) for personal accounts or [https://github.com/organizations/codemagic-ci-cd/settings/installations](https://github.com/organizations/codemagic-ci-cd/settings/installations) for organization accounts to check whether Codemagic CI/CD is listed both under installed and authorized GitHub Apps.
+* If you build apps from different GitHub accounts or organizations, make sure that you have installed the GitHub App on the account that has the repository available. Click **Manage integration > Configure** to see which accounts have been connected. You can also visit [https://github.com/settings/installations](https://github.com/settings/installations) for personal accounts or github(dot)com/organizations/your-organization/settings/installations for organization accounts to check whether Codemagic CI/CD is listed both under installed and authorized GitHub Apps.
```

Comment:
I would probably write this as 
https://github.com/organizations/YOUR_ORGANIZATION/settings/installations
but this is fine, it should be a short lived doc.